src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
```

Comment:
The `area` method in `IsoscelesTrapezoid` should not merely call `super.area()`. You should implement the logic to calculate the area of an isosceles trapezoid here, based on its specific properties.